system-test/bigquery.ts

```diff
@@ -646,6 +646,25 @@ describe('BigQuery', () => {
     });
   });
 
+  describe('BigQuery/Job', () => {
+    it('should delete a job', async () => {
+      const opts = {
+        configuration: {
+          query: {
+            query: 'SELECT 100 as foo',
+          },
+        },
+        location: 'us-east1',
+      };
+
+      const [job] = await bigquery.createJob(opts);
```

Comment:
It eoulf likely be worth adding some logic to cleanup stale jobs, kif this test were to fail mid-way we could leak some resources.

Is it possible to give the job an identifier, so that we could just list all jobs in a cleanup step, and delete any older than a certain age?

Reply:
for sure